stan/math/rev/mat/functor/adj_jac_apply.hpp

```diff
@@ -230,48 +333,107 @@ struct adj_jac_vari : public vari {
     prepare_x_vis(args...);
   }
 
+  /**
+   * The constructor initializes is_var_ with whether or not the scalar type in
```

Comment:
Just say `Initializes `is_var_` with true if the scalar type in each argument is a var.`